docs/tables/gcp_sql_database_instance_metric_connections_hourly.md

```diff
@@ -0,0 +1,55 @@
+# Table: gcp_sql_database_instance_metric_connections_hourly
+
```

Comment:
should it be gcp_sql_database_instance_connections_hourly

Reply:
@rajmohanty17 We have followed same naming pattern defined in AWS